JS/Lesson35/savas/my-app/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { NavLink, useMatch, useResolvedPath } from "react-router-dom";
+import "./navbar.css";
+
+const CustomLink = ({ to, pageName }) => {
+  const resolved = useResolvedPath(to);
+  const isActive = useMatch({ path: resolved.pathname });
+
+  return (
+    <li className={`item ${isActive ? "active" : ""}`}>
+      <NavLink to={to} exact={true} activeClassName="active">
```

Comment:
Please check the documentation for the NavLink component here: https://reactrouter.com/en/main/components/nav-link#navlink

You don't need to wrap NavLink into CustomLink component as NavLink gives you access to `isActive` parameter directly:
```
<NavLink
  to="/messages"
  className={({ isActive, isPending }) =>
    isPending ? "pending" : isActive ? "active" : ""
  }
>
  Messages
</NavLink>;
```